src/context/WalletProvider/Ledger/constants.ts

```diff
@@ -0,0 +1,36 @@
+import {
+  bchAssetId,
+  btcAssetId,
+  cosmosAssetId,
+  dogeAssetId,
+  ethAssetId,
+  fromAssetId,
+  ltcAssetId,
+  thorchainAssetId,
+} from '@shapeshiftoss/caip'
+import { uniq } from 'lodash'
+import { getSupportedEvmChainIds } from 'lib/utils/evm'
+
+/*
+  The top-level fee assets supported by Ledger, which can be mapped to a specific Ledger app.
+  The Ethereum app supports all EVM chains, so we don't need to list them all here.
+*/
+export const availableLedgerAppAssetIds = [
+  btcAssetId,
+  dogeAssetId,
+  bchAssetId,
+  ltcAssetId,
+  ethAssetId,
+  thorchainAssetId,
+  cosmosAssetId,
+]
```

Comment:
do we need to take into account EVMs that dont use ETH as the fee asset, e.g AVAX?

Reply:
I don't believe so, this maintains the current behaviour, which is for the Ethereum app to handle them all.